supertokens_python/recipe/thirdpartyemailpassword/interfaces.py

```diff
@@ -72,6 +90,73 @@ async def update_email_or_password(self, user_id: str, email: Union[str, None],
         pass
 
 
+class SignInUpPostOkResponse(APIResponse):
+    status: str = 'OK'
+
+    def __init__(self, user: User, created_new_user: bool,
+                 auth_code_response: Dict[str, Any],
+                 session: SessionContainer):
+        self.user = user
+        self.created_new_user = created_new_user
+        self.auth_code_response = auth_code_response
+        self.session = session
+
+    def to_json(self) -> Dict[str, Any]:
+        user: Dict[str, Any] = {
+            'id': self.user.user_id,
+            'email': self.user.email,
+            'timeJoined': self.user.time_joined,
+        }
+        if self.user.third_party_info is not None:
```

Comment:
Instead of doing `if`.. you should add an `assert` that third party info is not none